tests/test_ocs_basic_install.py

```diff
@@ -160,7 +160,7 @@ def run(**kwargs):
     create_oc_resource(
         "prometheus-rules.yaml", rook_data, cluster_path, _templating
     )
-
+    create_oc_resource('filesystem.yaml', rook_data, cluster_path, _templating)
```

Comment:
Lets move this in Rook Class as `def create_filesystem()`,  I think once I see the admin guide, we can revisit how to handle deployment with required services.